### PR TITLE
5007 set operations

### PR DESCRIPTION
**Pull request by leviathan747**
https://github.com/xtuml/bridgepoint/pull/579

### CODE REVIEW

File: test/test_set_operations_5007/gen/bridge.mark [hunk 1]
```diff
@@ -0,0 +1,59 @@
+.//============================================================================
```
Comment (cortlandstarrett) [inline test/test_set_operations_5007/gen/bridge.mark]: delete this file

File: test/test_set_operations_5007/gen/class.mark [hunk 2]
```diff
@@ -0,0 +1,250 @@
+.//============================================================================
```
Comment (cortlandstarrett) [inline test/test_set_operations_5007/gen/class.mark]: delete this file

File: test/test_set_operations_5007/gen/datatype.mark [hunk 3]
```diff
@@ -0,0 +1,166 @@
+.//============================================================================
```
Comment (cortlandstarrett) [inline test/test_set_operations_5007/gen/datatype.mark]: delete this file

File: test/test_set_operations_5007/gen/event.mark [hunk 4]
```diff
@@ -0,0 +1,43 @@
+.//============================================================================
```
Comment (cortlandstarrett) [inline test/test_set_operations_5007/gen/event.mark]: delete this file

File: test/test_set_operations_5007/models/test_set_operations_5007/lib/comp1/functions/.gitignore [hunk 5]
```diff
@@ -0,0 +1,2 @@
+/functions__foo.2.oal_err
```
Comment (cortlandstarrett) [inline test/test_set_operations_5007/models/test_set_operations_5007/lib/comp1/functions/.gitignore]: delete this file

Comment (leviathan747) [reply test/test_set_operations_5007/models/test_set_operations_5007/lib/comp1/functions/.gitignore]: done

File: test/test_set_operations_5007/gen/event.mark (hunk 4 above)
Comment (leviathan747) [reply test/test_set_operations_5007/gen/event.mark]: done

File: test/test_set_operations_5007/gen/bridge.mark (hunk 1 above)
Comment (leviathan747) [reply test/test_set_operations_5007/gen/bridge.mark]: done

**Pull request merged**